service/lib/agama/software/manager.rb

```diff
@@ -470,6 +470,27 @@ def import_gpg_keys
       end
 
       def add_base_repos
+        return if add_repos_by_label
+        return if add_repos_by_dir
+
+        # local repositories not found, use the online repositories
+        product.repositories.each { |url| repositories.add(url) }
+      end
+
+      def add_repos_by_dir
+        # path to cdrom which can contain installation repositories
+        dir_path = "/run/initramfs/live/install"
+
+        if File.exist?(dir_path)
```

Comment:
Just nit-picking, I'd reverse the logic:

```ruby
return false unless File.exist?(dir_path)

logger.info "/install found on installation medium"
repositories.add("dir://" + dir_path)
true
```

This avoids using an indented block.

Reply:
Yes, searching for a dir should be faster.

Reply:
@kobliha I think you mean change of order of checking for dir versus label?

Reply:
I do not think it will make difference and lets merge this one for now to have something working